core/sql/pom.xml.apache

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
```

Comment:
Rather than having multiple pom.xml files, could we use profiles in a single pom.xml? Just asking, I can imagine very well that it won't work.